lib/_constants.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  CLASS_NAMESPACE: 'gmusic-theme',
+  CLASS_NAMESPACE: 'gmusic-theme.material',
```

Comment:
Yeah, I mentioned that in the PR notes. Seeing as we're suffixing them all with !important anyway, it's probably not required.

I think I'm going to just strip it out.  If nothing else, it'll look a lot tidier.